utils/tinystr/Cargo.toml

```diff
@@ -41,7 +41,7 @@ criterion = "0.4"
 postcard = { version = "1.0.0", features = ["use-std"], default-features = false }
 rand = { version = "0.8.5", features = ["small_rng"] }
 serde_json = { version = "1.0", default-features = false, features = ["alloc"] }
-tinystr_old = { version = "0.4", package = "tinystr", features = ["serde"] }
+tinystr_old = { version = "0.7.1", package = "tinystr", features = ["serde"] }
```

Comment:
this shouldn't happen; but also I think we can safely remove this test now

Reply:
https://github.com/unicode-org/icu4x/pull/3326

This is the one "not just a version bump" lockfile change, perhaps once we undo that the windows build will work again

(I think it's possible we're quite close to a threshold and the minor lockfile change brings us over)